src/main/webapp/WEB-INF/templates/revoke-token.html

```diff
@@ -40,10 +46,11 @@
     <script>
       $(document).ready(function() {
 
-
-      	$('#revokeTokenButton').click(function() {
-      	
-      		console.log("Revoke Button Pressed");
+      	$('#revokeTokenButton').click(function() {      	
+			$('#errorMessage').hide();
+			$('#errorMessage').removeClass('bg-danger');
+			$('#errorMessage').removeClass('bg-info');
```

Comment:
I think you could, in theory, chain these methods (and other places that these get called in series), so jQuery doesn't have to do the selector 3 times in a row.
```suggestion
			$('#errorMessage').hide();
			                               .removeClass('bg-danger');
			                               .removeClass('bg-info');
```

Reply:
Chained it in multiple places in new commit!